fireant/slicer/filters.py

```diff
@@ -45,6 +56,11 @@ def __init__(self, element_key, values):
     def schemas(self, element):
         return element.notin(self.values)
 
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) \
+               and self.element_key == other.element_key \
+               and self.values == other.values
```

Comment:
@twheys Comparing a container with == is not really comparing its content/order. So this would only be true if the same values list (instance in memory) was used to initialise this filter.